### DISCUSSION TITLE
0A349D4D-858B-7EF3-7997-2FFE0DEA8313

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#82] [General]
# Quintuplet Cluster

This addon adds the Quintuplet Cluster, a cluster of hot and massive stars only a few hundred light years from the Milky Way's center. This addon includes more than 20 of the many stars of the Quintuplet Cluster, including the Pistol Star and its LBV relatives, V4650 Sgr and V4998 Sgr.

https://celestia.mobi/addon?item=0A349D4D-858B-7EF3-7997-2FFE0DEA8313&lang=zh_CN

**Comment by 13940951572** [comment]
这个天怎么不更新了软件
